src/go/k8s/config/crd/patches/re-add-checksum-field_in_gitrepositories.source.toolkit.fluxcd.io.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# The following patch adds the spec.versions.v1beta2.schema.openAPIV3Schema.properties.status.properties.artifact.properties.checksum property
+# and the spec.versions.v1beta2.schema.openAPIV3Schema.properties.status.properties.includedArtifacts.items.properties.checksum property
+# that was removed
+- op: add
+  path: /spec/versions/1/schema/openAPIV3Schema/properties/status/properties/artifact/properties/checksum
```

Comment:
I'm afraid of the numbers like `0` or `1`. If we change the order we will end up in the same situation.